src/vaults/PoolManager.sol

```diff
@@ -119,6 +115,7 @@ contract PoolManager is
         protected
     {
         IShareToken share = IShareToken(shareToken(poolId, scId));
+        require(centrifugeId != sender.localCentrifugeId(), LocalTransferNotAllowed());
```

Comment:
What happens if the receiver is in the same chain? Should be a normal ERC20 transfer?